README.md

```diff
@@ -6,56 +6,55 @@ TSMP2 workflow engine for running simulations. The following examples and descri
 
 ## Setup the workflow
 
+Activate a compute project
+```bash
+# Replace PROJECTNAME with your compute project
+jutil env activate -p PROJECTNAME
+
+# Check if $BUDGET_ACCOUNTS was set.
+echo $BUDGET_ACCOUNTS
+```
+
+In case you are not on a [JSC](https://www.fz-juelich.de/) machine, set the shell variables `BUDGET_ACCOUNT`, `PROJECT` and `SCRATCH` manually.
+Instead of setting `BUDGET_ACCOUNT` you may also replace this variable in `ctl/control_tsmp2.sh`
+
 ``` bash
-cd $PROJECT_DIR
-git clone --recurse-submodules https://github.com/HPSCTerrSys/TSMP2_workflow-engine
-wfe_dir=realpath tsmp2_workflow-engine
+cd $PROJECT/$USER
+git clone https://github.com/HPSCTerrSys/TSMP2_workflow-engine
+wfe_dir=$(realpath tsmp2_workflow-engine)
+git submodule init
+git submodule update
 ```
 
 ## Building the model
 
 The TSMP2 ( https://github.com/HPSCTerrSys/TSMP2 ) should be either already compiled (see [ReadMe TSMP2](https://github.com/HPSCTerrSys/TSMP2/blob/master/README.md)) or compiled with the following steps.
 
-0) change directory
 ```bash
 cd ${wfe_dir}/src/TSMP2
+./build_tsmp2.sh --tsmp2_env env/jsc.2024_Intel.sh --icon --eclm --parflow
 ```
 
-1) compile the code
-
-```bash
-# TSMP2
-./build_tsmp2.sh --icon --eclm --parflow
-```
+Adjust the components to your purpose.
 
 ## Run experiment
 
-Create and softlink run-directory on SCRATCH
+Create run-directory on SCRATCH
 ``` bash
-export SCRATCH_DIR=/p/scratch/YOUR_PROJECT/$USER/$sim_id
-mkdir -pv $SCRATCH_DIR
-ln -s $SCRATCH_DIR/run run
```

Comment:
Why you have removed the generation of the symlink? I would rather remove the run-directory than removing the softlink. In case of no generation of symlink, also the generation of the `$SCRATCH_DIR` would not be necessary.

Reply:
I readded it.

I don't understand why a `run` symlink must be created, i.e. why `run` is at two places, but back to that after the merge maybe.

Reply:
I understand now.  User is to remove the `run` directory with a git command as written in the README.

No `rm -f` needed, but of course it would go wrong if the user has some files in there, but she shouldn't.